tests/SprykerEcoTest/Zed/UnzerApi/_support/UnzerApiZedTester.php

```diff
@@ -0,0 +1,316 @@
+    public const CUSTOMER_COMPANY = 'Spryker';
+    public const CUSTOMER_SALUTATION = 'Mr';
+
+    public const METADATA_STORE = 'DE';
+    public const METADATA_PRICE_MODE = 'Gross';
+    public const METADATA_LOCALE = 'de';
+    public const PAYMENT_METHOD_SOFORT = 'sofort';
+
+    public const WEBHOOK_EVENT = 'all';
+    public const WEBHOOK_URL = 'https://unzer-spryker.com';
+
+    public const BASKET_ORDER_ID = 'spryker-order-id-2';
+    public const BASKET_NOTE = 'note';
+    public const BASKET_CURRENCY_CODE = 'EUR';
+    public const BASKET_AMOUNT_TOTAL_GROSS = 723.46;
+
+    const BASKET_ITEM_TYPE = 'Wire';
+    const BASKET_ITEM_PARTICIPANT_ID = '000000';
+    const BASKET_ITEM_TITLE = 'Canon Self-Shot';
+    const BASKET_ITEM_AMOUNT_PER_UNIT = 123.00;
+    const BASKET_ITEM_AMOUNT_GROSS = 123.00;
+    const BASKET_ITEM_REFERENCE_ID = 'spryker-sku-1';
+
+
+    /**
+     * @param \Codeception\Scenario $scenario
+     */
+    public function __construct(Scenario $scenario)
+    {
+        parent::__construct($scenario);
+
+        $this->setUpConfig();
+    }
+
+    /**
+     * @return void
+     */
+    protected function setUpConfig(): void
+    {
+        $this->setConfig(UnzerApiConstants::WEBHOOK_RESOURCE_URL, 'https://api.unzer.com/v1/webhooks');
+        $this->setConfig(UnzerApiConstants::CUSTOMER_RESOURCE_URL, 'https://api.unzer.com/v1/customers/%s');
+        $this->setConfig(UnzerApiConstants::BASKET_RESOURCE_URL, 'https://api.unzer.com/v1/baskets');
+        $this->setConfig(UnzerApiConstants::MARKETPLACE_AUTHORIZE_URL, 'https://api.unzer.com/v1/marketplace/payments/authorize');
+        $this->setConfig(UnzerApiConstants::AUTHORIZE_URL, 'https://api.unzer.com/v1/payments/authorize');
+        $this->setConfig(UnzerApiConstants::METADATA_RESOURCE_URL, 'https://api.unzer.com/v1/metadata');
+        $this->setConfig(UnzerApiConstants::MARKETPLACE_GET_PAYMENT_URL, 'https://api.unzer.com/v1/marketplace/payments');
+        $this->setConfig(UnzerApiConstants::GET_PAYMENT_URL, 'https://api.unzer.com/v1/payments/%s');
+        $this->setConfig(UnzerApiConstants::CHARGE_URL, 'https://api.unzer.com/v1/payments/charges');
+        $this->setConfig(UnzerApiConstants::MARKETPLACE_CHARGE_URL, 'https://api.unzer.com/v1/marketplace/payments/charges');
+        $this->setConfig(UnzerApiConstants::MARKETPLACE_CREDIT_CARD_CHARGE_URL, 'https://api.unzer.com/v1/marketplace/payments/%s/charges');
+        $this->setConfig(UnzerApiConstants::CREDIT_CARD_CHARGE_URL, 'https://api.unzer.com/v1/payments/%s/charges');
+        $this->setConfig(UnzerApiConstants::CREATE_PAYMENT_RESOURCE_URL, 'https://api.unzer.com/v1/types/%s');
+        $this->setConfig(UnzerApiConstants::MARKETPLACE_REFUND_URL, 'https://api.unzer.com/v1/marketplace/payments/%s/charges/%s/cancels');
+        $this->setConfig(UnzerApiConstants::REFUND_URL, 'https://api.unzer.com/v1/payments/%s/charges/%s/cancels');
+    }
+
+    /**
+     * @return UnzerApiConfig
```

Comment:
```suggestion
     * @return SprykerEco\Zed\UnzerApi\UnzerApiConfig
```

Please also check other places below